receiver/statsdreceiver/transport/udp_server.go

```diff
@@ -56,10 +59,8 @@ func (u *udpServer) ListenAndServe(
 			u.handlePacket(bufCopy, addr, transferChan)
 		}
 		if err != nil {
-			u.reporter.OnDebugf("UDP Transport (%s) - ReadFrom error: %v",
-				u.packetConn.LocalAddr(),
-				err)
 			var netErr net.Error
+			reporter.OnDebugf("%s Transport (%s) - ReadFrom error: %v", u.transport, u.packetConn.LocalAddr(), err)
```

Comment:
nit: now it is more readable but we probably didn't need to move the log line bellow `var netErr net.Error`

Reply:
True, might make confusion while doing the review. Changed back.